scripts/local_testnet/start_local_testnet.sh

```diff
@@ -139,7 +139,7 @@ sed -i 's/"shanghaiTime".*$/"shanghaiTime": 0,/g' $genesis_file
 for (( bn=1; bn<=$BN_COUNT; bn++ )); do
     secret=$DATADIR/geth_datadir$bn/geth/jwtsecret
     echo $secret
-    execute_command_add_PID beacon_node_$bn.log ./beacon_node.sh $SAS -d $DEBUG_LEVEL $DATADIR/node_$bn $((BN_udp_tcp_base + $bn)) $((BN_http_port_base + $bn)) http://localhost:$((EL_base_auth_http + $bn)) $secret
+    execute_command_add_PID beacon_node_$bn.log ./beacon_node.sh $SAS -d $DEBUG_LEVEL $DATADIR/node_$bn $((BN_udp_tcp_base + $bn)) $((BN_http_port_base + $bn)) $((BN_udp_tcp_base + $bn + 100)) http://localhost:$((EL_base_auth_http + $bn)) $secret
```

Comment:
This change also needs to be made in the `doppelganger_protection` script as it uses the same `beacon_node.sh` script.

https://github.com/sigp/lighthouse/blob/70c4ae35aba72e86884143ba054012e104008c04/scripts/tests/doppelganger_protection.sh#L58-L60